src/base64.cr

```diff
@@ -149,11 +149,8 @@ module Base64
   # Alphabet" in RFC 4648.
   #
   # The alphabet uses '-' instead of '+' and '_' instead of '/'.
-  #
-  # The `padding` parameter defaults to false. When true, enough `=` characters
-  # are added to make the output divisible by 3.
   def urlsafe_encode(data, io : IO)
```

Comment:
i want to remove padding option, and i thought this is not needed, but used in secure_random
https://github.com/crystal-lang/crystal/blob/master/src/secure_random.cr#L46

in io method, i not think it needed